### PR TITLE
Gun names (again)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Changed a few gun names 

-L6 SAW and Gyrojet because they had strange redundant names (SAW LMG triggered me)

-FK-69 and MLG-12 because they had meme names

-Guns with proper names like the mosin nagant, desert eagle, lost "pistol" and stuff at the end. I think it sounds akward.

-Changed captains laser name for zero reason other than I thought the random numbers/letters were a bad name.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin"
```
Comment (Iamgoofball) [inline code/modules/projectiles/guns/projectile/pistol.dm]: the m1911 and deagle are explanatory by themselves, could pistol at least be tacked on to the end of this one?

**Comment by Xhuis** [comment]
You might want to change "stetchkin" to "stetchkin pistol". Just personal feedback tho

**Comment by Iamgoofball** [comment]
5d399ade-30a6-11e5-93e1-fdfbdf35ce9e

**Comment by Xhuis** [comment]
# ___THAT'S MY MEME___

**Comment by PKPenguin321** [comment]
>FK-69

who let that get merged in the first place

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
-	name = "\improper FK-69 pistol"
+	name = "stechkin pistol"
```
Comment (bear1ake) [inline code/modules/projectiles/guns/projectile/pistol.dm]: :-1:

Comment (Iamgoofball) [inline code/modules/projectiles/guns/projectile/pistol.dm]: why

Comment (bear1ake) [inline code/modules/projectiles/guns/projectile/pistol.dm]: because this pistol isn't stechkin

Comment (Iamgoofball) [inline code/modules/projectiles/guns/projectile/pistol.dm]: The sprite is actually closer to a walther ppk

Comment (bear1ake) [inline code/modules/projectiles/guns/projectile/pistol.dm]: And... Walther PPK have a 10mm modification!
Why we don't rename it to Walther?

Comment (Dorsisdwarf) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Because fuck you it was called a stetchkin for at least three years before anyone even realised there was an actual gun called a stetchkin, you /k/ fuck.

**Comment by Dorsisdwarf** [comment]
If you're renaming the m12g bulldog (Why was it renamed the MLG?!), you should repath the ammo and mags to "bulldog" so its easier for badmins to spawn it and so on.

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 3]
```diff
@@ -16,7 +16,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/pistol/m1911
-	name = "\improper M1911 pistol"
+	name = "M1911"
```
Comment (Firecage) [inline code/modules/projectiles/guns/projectile/pistol.dm]: This should keep pistol at the end. That was done EXACTLY for the reason so you could easily see what type of gun it is or what the heck was shot at you.

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 4]
```diff
@@ -24,7 +24,7 @@
 	can_suppress = 0
 
 /obj/item/weapon/gun/projectile/automatic/pistol/deagle
-	name = "\improper Desert Eagle pistol"
+	name = "desert eagle"
```
Comment (Firecage) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Same as I said above.

File: code/modules/projectiles/guns/projectile/shotgun.dm [hunk 5]
```diff
@@ -85,7 +85,7 @@
 ///////////////////////
 
 /obj/item/weapon/gun/projectile/shotgun/boltaction
-	name = "\improper Mosin Nagant rifle"
+	name = "\improper Mosin Nagant"
```
Comment (Firecage) [inline code/modules/projectiles/guns/projectile/shotgun.dm]: Same as above.

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 6]
```diff
@@ -136,7 +136,7 @@
 	burst_size = 2
 
 /obj/item/weapon/gun/projectile/automatic/l6_saw
-	name = "\improper L6 SAW LMG"
+	name = "\improper L6 SAW"
```
Comment (Firecage) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Same as above. keep the LMG at the end.

Comment (KorPhaeron) [inline code/modules/projectiles/guns/projectile/automatic.dm]: SAW is the type of gun though. Your name was "L6 squad automatic support weapon light machine gun"

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 2 above)
Comment (Cheridan) [inline code/modules/projectiles/guns/projectile/pistol.dm]: more like it was noticed a lot earlier than that but nobody bothered to change it

Comment (KorPhaeron) [inline code/modules/projectiles/guns/projectile/pistol.dm]: I swear to god I'm just going to go back to "syndicate pistol" and have a description about a stolen design of forgotten origin.

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 3 above)
Comment (freerealestate) [inline code/modules/projectiles/guns/projectile/pistol.dm]: yeah, it's inconsistent with the other naming changes in the pull

not everyone knows that M1911 is a 9mm pistol

Comment (KorPhaeron) [inline code/modules/projectiles/guns/projectile/pistol.dm]: It has the pistol in hand sprite. It makes a loud bang noise. It sends bullets at you. What is confusing about that?

Comment (freerealestate) [inline code/modules/projectiles/guns/projectile/pistol.dm]: isn't that applicable to a lot of different guns, though? not just pistols? a lot of them share the same in-hand, the same gunshot sound.

why do you want this to be called m1911 only, while being happy to call the stetchkin a pistol and just name the captain's gun a laser gun?

**Comment by optimumtact** [comment]
I think that needing to identify exactly what type of gun is shooting at you is a pretty silly reason to put rifle/pistol on the end of the gun name.

99% of the time you only care about the fact they're firing a gun at you, not what exact type of gun it might be.

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/modules/projectiles/guns/projectile/pistol.dm]: @Dorsisdwarf 
I'm 99% certain that it used to have an actual stetchkin sprite mport lifted from google images or some shit and scaled down like the deagle sprites

the resprite of everything probably made it a walther ppk

**Comment by Lumipharon** [comment]
Fuck no to sticking pistol, rifle, smg or whatever fuck at the end of the names.

There is no functional difference between what 'pistols' or 'rifles' etc do, only what specific guns do.

**Comment by freerealestate** [comment]
Why's this argument exist, anyway?

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 3 above)
Comment (phil235) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Why did you remove the \improper ?

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 6 above)
Comment (Firecage) [inline code/modules/projectiles/guns/projectile/automatic.dm]: I thought it was semi-automatic weapon?

Comment (whodaloo) [inline code/modules/projectiles/guns/projectile/automatic.dm]: in what universe is a LMG semi automatic

**Comment by tuypo1** [comment]
because we are the sort of people that will argue over how much dirt a space station should have and this is a much bigger deal

**Comment by SconesC** [comment]
These names were retarded in the first place and just look like a 6 year old who started browsing /k/ last week wanted to leave his mark on the game

**Comment by freerealestate** [comment]
^^^

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 2 above)
Comment (Cheridan) [inline code/modules/projectiles/guns/projectile/pistol.dm]: @Iamgoofball 
You're completely wrong

**Comment by Dorsisdwarf** [comment]
The only shitty names are the shotguns, HOS gun, captain gun, "MLG" (It's M12g damnit) bulldog, and the Cheridan'sAButt pistol

**Comment by freerealestate** [comment]
so basically all of them

**Comment by Bawhoppen** [comment]
I still think that 'pistol' and 'rifle' and stuff should be kept at the end of the name

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 3 above)
Comment (phil235) [inline code/modules/projectiles/guns/projectile/pistol.dm]: @KorPhaeron I needs the \improper since the name is capitalized.

Comment (KorPhaeron) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Whoops I'll fix it

**Comment by KorPhaeron** [comment]
@phil235 

It's fixed

**Comment by phil235** [comment]
Closing this one and merging the one with less commits
